content/docs/nav.yml

```diff
@@ -36,37 +36,37 @@
     - id: composition-vs-inheritance
       title: Composition vs Inheritance
     - id: thinking-in-react
-      title: Thinking In React
+      title: Tư duy trong React
 - title: Hướng dẫn nâng cao
   items:
     - id: accessibility
-      title: Accessibility
+      title: Khả năng tiếp cận
     - id: code-splitting
       title: Code-Splitting
     - id: context
       title: Context
     - id: error-boundaries
       title: Error Boundaries
     - id: forwarding-refs
-      title: Forwarding Refs
+      title: Chuyển tiếp Refs
     - id: fragments
       title: Fragments
     - id: higher-order-components
       title: Higher-Order Components
     - id: integrating-with-other-libraries
-      title: Integrating with Other Libraries
+      title: Tích hợp với các thư viện khác
     - id: jsx-in-depth
       title: JSX In Depth
     - id: optimizing-performance
-      title: Optimizing Performance
+      title: Tối ưu hiệu năng
     - id: portals
       title: Portals
     - id: profiler
       title: Profiler
     - id: react-without-es6
-      title: React Without ES6
+      title: React không sử dụng ES6
```

Comment:
React "nhưng" không sử dụng ES6

Do câu trên đọc sợ người đọc hiểu lầm là Reactjs ko có sử dụng ES6 toàn bộ luôn ấy